templates/zerver/integrations/include/event-filtering-additional-feature.md

```diff
@@ -0,0 +1,16 @@
+* The {{ integration_display_name }} integration supports filtering for
+  the following events:
+
+    {% set comma = joiner(", ") %}
+
+    {% for event_type in all_event_types -%} {{- comma() -}} `{{ event_type }}` {%- endfor %}
+
+    To filter the events that trigger the notifications, you can append
```

Comment:
When the events filtering is possible via the modal, then from here below could be moved to the webhooks URL specification documentation.

Reply:
Yeah I think that'd be nice.